public/app/features/dashboard/dashgrid/DashboardEmpty.tsx

```diff
@@ -26,13 +27,13 @@ const DashboardEmpty = ({ dashboard, canCreate }: Props) => {
     <div className={styles.centeredContent}>
       <div className={cx(styles.centeredContent, styles.wrapper)}>
         <div className={cx(styles.containerBox, styles.centeredContent, styles.visualizationContainer)}>
-          <div className={styles.headerBig}>
+          <Box marginBottom={2}>
```

Comment:
this change breaks this design (changes background). and I don't understand the use case here really. No need for a box of any kind, just Flex with some padding.  What I mean this div should not be needed really, with proper use of flex and gap

Reply:
Really? 🤔  I don't see any change between what's on play and what's on my local, I don't even see these divs with any background, could you share a screenshot of where this happens?

Reply:
try using light theme :)

Reply:
bug should happen on dark theme as well, the Box component defaults to primary background, and this box here should not have a background (it's on canvas).

Reply:
yea, see the same bug in both light and dark theme. Maybe the contrast on your monitor is too low :)

Reply:
Ok something was really wrong with my build, because I couldnt see it and then I made some changes, it rebuilt and now its glaringly obvious 😂 

We can make `backgroundColor` have no default for this anyway 👍